doc/conf.py

```diff
@@ -69,7 +69,7 @@
 
 # Switch to old behavior with html4, for a good display of references,
 # as described in https://github.com/sphinx-doc/sphinx/issues/6705
-html4_writer = True
+# html4_writer = True
```

Comment:
If this is no longer needed, can we delete it?

Same goes for default_role above.

Reply:
Yes. Deleted it